Content.Server/GameObjects/Components/Access/AccessReaderComponent.cs

```diff
@@ -47,8 +47,9 @@ private bool IsAllowed(List<string> accessTags)
                 {
                     return false;
                 }
+                return true;
```

Comment:
This makes it impossible to have airlocks with multiple necessary tags since the first will always complete.

Reply:
Since i couldnt find how to fix this i just gave the EVAs airlocks External access as necessary. Im going to make an issue about this.